packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/get.js

```diff
@@ -883,6 +883,30 @@ describe('plugin-conversation', function () {
             assert.isTrue(actCount < minActivities);
           }));
 
+        it('should return all activities in a space when jumping to an activity in a space with more activities than the asked-for activities limit', () => webex.internal.conversation.create({participants: [scott.id]})
+          .then((c) => webex.internal.conversation.post(c, {displayName: 'message 1'})
+            .then(() => webex.internal.conversation.post(c, {displayName: 'message 2'})
+              .then(() => webex.internal.conversation.post(c, {displayName: 'message 3'})
+                .then(() => webex.internal.conversation.post(c, {displayName: 'message 4'})
+                  .then(() => webex.internal.conversation.post(c, {displayName: 'message 5'})
+                    .then(() => webex.internal.conversation.post(c, {displayName: 'message 6'})
+                      .then(() => webex.internal.conversation.post(c, {displayName: 'message 7'})
+                        .then(() => webex.internal.conversation.post(c, {displayName: 'message 8'})
+                          .then(() => webex.internal.conversation.post(c, {displayName: 'message 9'})
+                            .then(() => webex.internal.conversation.post(c, {displayName: 'message 10'})
+                              .then(() => webex.internal.conversation.post(c, {displayName: 'message 11'})
+                                .then(() => webex.internal.conversation.post(c, {displayName: 'message 12'})
+                                  .then(() => webex.internal.conversation.post(c, {displayName: 'message 13'})
```

Comment:
Could this be collapsed into a `for` block to send all the messages instead of a giant nesting block? 

```js
  const results = [];
  for (i = 0; i < 13; i++) {
    // Good: all asynchronous operations are immediately started.
    results.push(webex.internal.conversation.post(c, {displayName: `message ${i}`}));
  }
  // Now that all the asynchronous operations are running, here we wait until they all complete.
  return Promise.all(results));
}
```

Reply:
I have no problem with that. Change incoming.